configs/example/gem5_library/memory_traffic.py

```diff
@@ -99,6 +99,9 @@ def generator_factory(
 # We use the Test Board. This is a special board to run traffic generation
 # tasks. Can replace the cache_hierarchy with any hierarchy to simulate the
 # cache as well as the memory
+
+# Need to test this; will this work with cache_hierarchy as None instead of
+# NoCache?
```

Comment:
The test board should know how to deal with `None` cache instead of `NoCache`.  `NoCache` is actually a cache hierarchy, but that doesn't have any caches. `None` actually means that the generators are directly connected to the memory.